rotkehlchen/tests/api/test_current_assets_price.py

```diff
@@ -417,3 +417,62 @@ def test_get_all_current_prices(rotkehlchen_api_server):
     )
     result = assert_proper_response_with_result(response)
     assert len(result) == 0
+
+
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
```

Comment:
Why not mocking the prices here? This saves a few external requests that are not needed

Reply:
I think it's necessary. In this case, prices are not known at that point.

Reply:
The reason to not mock them is because the inquierer doesn't allow to mock prices only for certain oracles so if we mock them we invalidate the logic being tested here

Reply:
So maybe @prettyirrelevant can you leave a mention in the docstring explaining why prices are not mocked here?